emeis/core/migrations/0001_initial.py

```diff
@@ -132,7 +132,11 @@ class Migration(migrations.Migration):
                 ),
             ],
             options={"verbose_name": "user", "verbose_name_plural": "users"},
-            bases=(emeis.core.models.VisibilityMixin, models.Model),
+            bases=(
+                emeis.core.models.VisibilityMixin,
+                emeis.core.models.PermissionMixin,
```

Comment:
I guess the PermissionMixin was forgotten in the previous MR, right?

Reply:
Exactly.